build/templates/values.yaml

```diff
@@ -349,6 +349,10 @@ serviceMonitor:
   # Limits the ServiceMonitor to the current namespace if set to `true`.
   namespaced: false
 
+  # tlsConfig: TLS configuration to use when scraping the endpoint.
+  # Of type: https://github.com/coreos/prometheus-operator/blob/main/Documentation/api.md#tlsconfig
+  tlsConfig: {}
+
```

Comment:
You have to add this change in the cockroachdb/values.yaml file as well to make the tests pass.

Reply:
Thank you, @prafull01!